.github/ISSUE_TEMPLATE/technical-question.md

```diff
@@ -11,12 +11,14 @@ assignees: ''
 
 Thank you for asking a technical question! If your question is "why doesn't my code work?", please see [submit a bug report](https://github.com/Azure/azure-iot-sdk-java/issues/new?assignees=&labels=bug&template=bug-report.md&title=) instead.
 
-We encourage you to submit technical questions related to the SDK on Stack Overflow. We actively monitor all questions tagged with "azure-iot-sdk".
+For quick and reliable answers on your technical product questions from Microsoft Engineers, Azure Most Valuable Professionals (MVPs), or our expert community, engage with us on [Microsoft Q&A](https://aka.ms/azureqa), Azure’s preferred destination for community support. Please tag your question with "azure-iot-sdk".
```

Comment:
I wouldn't capitalize `Engineers`. It isn't a proper noun.

Reply:
yes good call, copied and pasted from [this page](https://docs.microsoft.com/en-us/azure/iot-fundamentals/iot-support-help) so should get a quick fix there too